e2e/live/end-to-end-test.sh

```diff
@@ -878,6 +878,38 @@ ${BIN_DIR}/kpt live apply e2e/live/testdata/continue-on-error > $OUTPUT_DIR/stat
 assertCMInventory "test-namespace" "1"
 assertPodExists "pod-a" "test-namespace"
 assertPodNotExists "pod-B" "test-namespace"
+
+# Test 21: RBAC error applying a resource
+echo "Testing RBAC error during apply"
+echo "kpt live apply e2e/live/testdata/rbac-error-step-1"
+echo "kpt live apply e2e/live/testdata/rbac-error-step-2"
+# Setup: create a service account and bind a Role to it so it has administrative
+# privileges on the "test" namespace, but no permissions on the default
+# namespace.
+kubectl apply -f e2e/live/testdata/rbac-error-step-1 > $OUTPUT_DIR/status
+assertContains "namespace/test created"
+assertContains "rolebinding.rbac.authorization.k8s.io/admin created"
+assertContains "serviceaccount/user created"
+wait 2
```

Comment:
I'm not sure I understand what this `wait` is doing. Can you add a comment here to make it easier to understand?

Reply:
We may not need the `wait`, but it helps insure there is sufficient time for the resources to actually be spun up on the cluster (e.g `namespace/test`) after they've been created in the API Server. The best way to do this is to do a `kubectl wait for=<CONDITION>`, but this is a shortcut. Checkout the bash code for `assertPodExists` and `assertPodNotExists` which uses `kubectl wait ...`.

Reply:
I was going to suggest `kubectl wait` depending on the use case, but I think you may be intending to [`sleep 2`](https://linux.die.net/man/3/sleep) instead of [`wait 2`](https://linux.die.net/man/3/wait) here. `wait` is going to delay until process id 2 exits. I'm not sure what `pid 2` refers to in this case or if that's what was intended? Maybe I'm misunderstanding?

Reply:
The `wait` is a bash function defined earlier in the script which calls `sleep`. The number is the number of seconds to sleep.